### PR TITLE
format translations for auto translate script

### PR DESCRIPTION
**Pull request by matthewthomsonnz**
TLDR: this PR just re-indents all the language json files

- We have now [merged a script into core-template](https://github.com/shift72/core-template/pull/294) that automatically translates a key for each translation file
- A side effect of this was all translations had to be auto formatted as it [rewrites the JSON with two indents](https://github.com/shift72/core-template/blob/develop/scripts/translate.mjs#L39)
- So @stajs [requested a PR](https://github.com/shift72/core-template/pull/295#issuecomment-1076942648) that does the formatting in one fell swoop
- Should theoretically be no different apart from intents/newlines, but cast your eyes over it just in case

### CODE REVIEW

**Review by stajs — COMMENTED** [review]
Happy to merge on faith once the conflict is resolved.
